site/docs/pt/guide/api.md

```diff
@@ -0,0 +1,136 @@
+# Bot API
+
+## Informações Gerais
+
+Os bots do Telegram se comunicam com os servidores do Telegram por meio de solicitações HTTP.
+A API de bots do Telegram é a especificação dessa interface, ou seja, uma [longa lista](https://core.telegram.org/bots/api) de métodos e tipos de dados, comumente chamada de referência.
```

Comment:
```suggestion
A Bot API do Telegram é a especificação dessa interface, ou seja, uma [longa lista](https://core.telegram.org/bots/api) de métodos e tipos de dados, comumente chamada de referência.
```

Reply:
In this case I think we can keep it as-is, because we're specifying that it's Telegram's Bot API